classes/CPU.js

```diff
@@ -183,7 +186,17 @@ class CPU {
 
                 break;
             case 'ADD_Vx_B':
```

Comment:
this needs to be changed to an uppercase X like decode,

'ADD_VX_B'

Reply:
I made the change